step.yml

```diff
@@ -237,35 +265,47 @@ inputs:
         Max time a test execution is allowed to run before it is automatically canceled. The default value is 900 (15 min).  Duration in seconds with up to nine fractional digits. Example: "3.5".
       description: |
         Max time a test execution is allowed to run before it is automatically canceled. The default value is 900 (15 min).  Duration in seconds with up to nine fractional digits. Example: "3.5".
-  - directories_to_pull:
+  - num_flaky_test_attempts: 0
```

Comment:
Note: this is perfectly fine, but FYI steps can have ranges as constraint (https://github.com/bitrise-io/go-steputils/blob/master/stepconf/stepconf.go#L30)